NuGetBuildValidators/common.ps1

```diff
@@ -0,0 +1,100 @@
+
+function Format-TeamCityMessage([string]$Text) {
+    $Text.Replace("|", "||").Replace("'", "|'").Replace("[", "|[").Replace("]", "|]").Replace("`n", "|n").Replace("`r", "|r")
+}
+
+Function Trace-Log($TraceMessage = '') {
+    Write-Host "[$(Trace-Time)]`t$TraceMessage" -ForegroundColor Cyan
+}
+
+Function Verbose-Log($VerboseMessage) {
+    Write-Verbose "[$(Trace-Time)]`t$VerboseMessage"
+}
+
+Function Error-Log {
+    param(
+        [string]$ErrorMessage,
+        [switch]$Fatal)
+    if (-not $Fatal) {
+        Write-Error "[$(Trace-Time)]`t$ErrorMessage"
+    }
+    else {
+        Write-Error "[$(Trace-Time)]`t[FATAL] $ErrorMessage" -ErrorAction Stop
+    }
+}
+
+Function Warning-Log($WarningMessage) {
+    Write-Warning "[$(Trace-Time)]`t$WarningMessage"
+}
+
+Function Trace-Time() {
+    $currentTime = Get-Date
+    $lastTime = $Global:LastTraceTime
+    $Global:LastTraceTime = $currentTime
+    "{0:HH:mm:ss} +{1:F0}" -f $currentTime, ($currentTime - $lastTime).TotalSeconds
+}
+
+$Global:LastTraceTime = Get-Date
+
+Function Format-ElapsedTime($ElapsedTime) {
+    '{0:F0}:{1:D2}' -f $ElapsedTime.TotalMinutes, $ElapsedTime.Seconds
+}
+
+Function Invoke-BuildStep {
```

Comment:
you don't need the teamcity stuff anymore

Reply:
Build scripts need updating